encodings/fastlanes/src/bitpacking/compute/is_constant.rs

```diff
@@ -16,8 +16,11 @@ impl IsConstantKernel for BitPackedVTable {
     fn is_constant(
         &self,
         array: &BitPackedArray,
-        _opts: &IsConstantOpts,
+        opts: &IsConstantOpts,
     ) -> VortexResult<Option<bool>> {
+        if opts.is_constant() {
```

Comment:
Hmmmm, I changed my mind. I think we should call this "Trivial", it's way too confusing otherwise!

Reply:
Yeah that's an improvement. I would still argue that "constant_<anything>" is more confusing that not using the word "constant". Immediate? Negligible?